git_machete/client.py

```diff
@@ -1950,25 +1950,28 @@ def __get_valid_pull_requests(prs_list: Optional[List[int]],
                         if len(prs_list) > 1:
                             warn(f"PR #{pr_no} is not found in repository `{org}/{repo}`, skipping.")
```

Comment:
I wouldn't make a difference in handling of the `len(prs_list) > 1` case, let's fail this one as well